docs/user/visualize.asciidoc

```diff
@@ -7,74 +7,79 @@ _Visualize_ enables you to create visualizations of the data from your {es} indi
+* *Data table* &mdash; Flattens aggregations into table format.
 
-Goal and gauge:: Display a number with progress indicators.
+* *Metric* &mdash; Display a single number.
 
-Tag cloud:: Display words in a cloud, where the size of the word corresponds to its importance.
+* *Goal and gauge* &mdash; Display a number with progress indicators.
+
+* *Tag cloud* &mdash; Display words in a cloud, where the size of the word corresponds to its importance.
 
-* *Time series optimized*
-[horizontal]
 <<TSVB,TSVB>>:: Visualize time series data using pipeline aggregations.
 
 <<timelion,Timelion>>:: Compute and combine data from multiple time series
 data sets.
 
-* *Maps*
-[horizontal]
-<<maps,Elastic Maps>>:: The most powerful way of visualizing map data in {kib}.
+<<visualize-maps, Maps>>::
+* *<<maps,Elastic Maps>>* &mdash; The most powerful way of visualizing map data in {kib}.
+
+* *Coordinate map* &mdash; Displays points on a map using a geohash aggregation.
 
-<<visualize-maps,Coordinate map>>:: Displays points on a map using a geohash aggregation.
+* *Region map* &mdash; Merge any structured map data onto a shape.
 
-<<visualize-maps,Region map>>:: Merge any structured map data onto a shape.
+* *Heat map* &mdash; Display shaded cells within a matrix.
 
-<<visualize-maps,Heat map>>:: Display shaded cells within a matrix.
+<<for-dashboard,Dashboard tools>>::
 
-* *<<for-dashboard,Dashboard tools>>*
-[horizontal]
-Controls:: Adds interactive inputs to a Dashboard.
+* *Markdown widget* &mdash; Display free-form information or instructions.
 
-Markdown widget:: Display free-form information or instructions.
+* *Controls* &mdash; Add interactive inputs to a Dashboard.
 
-* *For developers*
-[horizontal]
 <<vega-graph,Vega>>:: Complete control over query and display.
 
-. Specify a search query to retrieve the data for your visualization:
-** To enter new search criteria, select the <<index-patterns, index pattern>> for the indices that
+[float]
+[[choose-your-data]]
+== Choose your data
+
+Specify a search query to retrieve the data for your visualization, or used rolled up data.
+
+* To enter new search criteria, select the <<index-patterns, index pattern>> for the indices that
 contain the data you want to visualize. The visualization builder opens
 with a wildcard query that matches all of the documents in the selected
 indices.
-** To build a visualization from a saved search, click the name of the saved
+
+* To build a visualization from a saved search, click the name of the saved
 search you want to use. The visualization builder opens and loads the
 selected query.
 +
 NOTE: When you build a visualization from a saved search, any subsequent
-modifications to the saved search are automatically reflected in the
-visualization. To disable automatic updates, you can disconnect a visualization
+modifications to the saved search are reflected in the
+visualization. To disable automatic updates, disconnect the visualization
```

Comment:
Maybe we can be more specific here.  I believe that disconnect the visualization means to remove the link.